streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -111,8 +111,8 @@ public StreamTask(TaskId id,
         // initialize the topology with its own context
         this.processorContext = new ProcessorContextImpl(id, this, config, recordCollector, stateMgr, metrics);
 
-        // initialize the state stores
-        initializeStateStores();
+        // register the state stores
+        registerStateStores();
```

Comment:
We still call `((ProcessorContextImpl) this.processorContext).initialized();` in line 127 below, and we check `initialized` in `initStore`, would that be a problem?